### DISCUSSION TITLE
v2.0.5

### DISCUSSION THREAD

**Discussion by aaronhelton** [dag-hammarskjold-library/dlx-rest#1069] [General]
## What's Changed
* Re-addresses the duplicate authority checking bug identified in #1003 (fields with $g vs fields without) by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1059
* Fixes a bug that prevented search results from appearing if speeches lacked 700$a by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1060
* Improves loading time of history records when the list of events is very long by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1061
* Adds exception handling inside threads to improve error detection; Adds record snapshot to history prior to first save in ME so history is accurate (records imported from Horizon and not yet modified have no history yet) by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1062
* Updates 100$g to require parentheses by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1064
* make speeches fields 269 and 930 not required by @aaronhelton in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1068

**Full Changelog**: https://github.com/dag-hammarskjold-library/dlx-rest/compare/v2.0.4...v2.0.5

<hr /><em>This discussion was created from the release <a href='https://github.com/dag-hammarskjold-library/dlx-rest/releases/tag/v2.0.5'>v2.0.5</a>.</em>

**Comment by aaronhelton** [comment]
Deployed to QAT 2023-05-11

**Reply by viola-v** [reply]
Tested 2023-05-11 and reopened issue #1058
